### DISCUSSION TITLE
Localized Date(-time) Picker

### DISCUSSION THREAD

**Discussion by loranger** [filamentphp/filament#619] [Ideas]
Currently, Date picker and Date-time picker uses *PHP date formatting tokens* which displays dates in english only and does not use locales. We can change the [date format](https://i.redd.it/5pu48jqka4621.png), but the customization is very limited.

Maybe those pickers could use [strftime](https://www.php.net/manual/fr/function.strftime.php) tokens or Carbon as we all love and uses Laravel ?

**Comment by danharrin** [comment]
We have localisation support already I think. Are you using the forms or the admin package?

**Comment by loranger** [comment]
My bad, I had to dig a little deeper instead of trying to use carbon formatting. I was confused by the php date tokens which are not supposed to use locales where filament does.

Sorry for the noise
